### PR TITLE
ADCM-3131 `selector` generation fixed, `selector` format fixed

### PR DESCRIPTION
**Pull request by a-alferov**

### CODE REVIEW

File: python/api/job/serializers.py [hunk 1]
```diff
@@ -265,29 +187,29 @@ class LogStorageSerializer(serializers.Serializer):
...
+            config.RUN_DIR, f"{obj.job.id}", f"{obj.name}-{obj.type}.{obj.format}"
         )
         try:
-            with open(path_file, 'r', encoding='utf_8') as f:
+            with open(path_file, "r", encoding="utf_8") as f:
                 content = f.read()
         except FileNotFoundError:
             msg = f'File "{obj.name}-{obj.type}.{obj.format}" not found'
-            raise AdcmEx('LOG_NOT_FOUND', msg) from None
+            raise AdcmEx("LOG_NOT_FOUND", msg) from None
```
Comment (winmasta) [inline python/api/job/serializers.py:197]:
```suggestion
except FileNotFoundError as e:
            raise AdcmEx("LOG_NOT_FOUND", msg) from e
```

File: python/cm/issue.py [hunk 2]
```diff
@@ -25,85 +26,83 @@
     ConfigLog,
     Host,
     HostComponent,
-    MessageTemplate as MsgTpl,
-    PrototypeEnum,
-    Prototype,
-    PrototypeImport,
 )
+from cm.models import MessageTemplate as MsgTpl
```
Comment (winmasta) [inline python/cm/issue.py:30]: What is the purpose of alias ?

File: python/cm/migrations/0092_auto_20220906_0957.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                cluster=cluster,
+                service=service,
+                prototype=component_prototype,
+            )
+            context["service"] = {"id": service.pk, "name": service.prototype.display_name}
+            context["component"] = {
+                "id": component.pk,
+                "name": component.prototype.display_name,
+            }
+    except ObjectDoesNotExist:
```
Comment (winmasta) [inline python/cm/migrations/0092_auto_20220906_0957.py:71]: `try` block should be as short as possible, it this case we can divide it to some `try` blocks

File: python/cm/migrations/0092_auto_20220906_0957.py [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+    for task in TaskLog.objects.all():
+
+        if task.selector:
+            selector = _fix_selector(task.selector, _models)
+            task.selector = selector
+            task.joblog_set.filter().update(selector=selector)
+
+        else:
+            try:
+                model, _ = _models.get(MODEL_TYPE[task.object_type.model], (None, None))
```
Comment (winmasta) [inline python/cm/migrations/0092_auto_20220906_0957.py:112]: Can be out of `try` checked by `if ... not in ...: continue`

File: python/rbac/endpoints_ui/role/views.py [hunk 5]
```diff
@@ -14,7 +14,7 @@
 
 from collections import defaultdict
 
-from rest_framework import serializers, mixins
+from rest_framework import mixins, serializers
```
Comment (winmasta) [inline python/rbac/endpoints_ui/role/views.py:17]: Explicit import is better

Comment (a-alferov) [reply python/rbac/endpoints_ui/role/views.py:17]: Fixed

File: python/cm/migrations/0092_auto_20220906_0957.py (hunk 4 above)
Comment (a-alferov) [reply python/cm/migrations/0092_auto_20220906_0957.py]: Fixed

File: python/cm/migrations/0092_auto_20220906_0957.py (hunk 3 above)
Comment (a-alferov) [reply python/cm/migrations/0092_auto_20220906_0957.py]: Fixed

File: python/cm/issue.py (hunk 2 above)
Comment (a-alferov) [reply python/cm/issue.py]: Fixed

File: python/api/job/serializers.py (hunk 1 above)
Comment (a-alferov) [reply python/api/job/serializers.py]: Fixed

File: python/rbac/endpoints_ui/role/views.py (hunk 5 above)
Comment (a-alferov) [reply python/rbac/endpoints_ui/role/views.py]: Please look again

**Review by winmasta — APPROVED** [review]

**Review by winmasta — APPROVED** [review]

**Review by winmasta — APPROVED** [review]

**Pull request merged**